defaults/main.yml

```diff
@@ -71,13 +71,22 @@ omero_server_python_addons: []
 # If non-empty dump the OMERO database to this directory before upgrading
 omero_server_database_backupdir: ""
 
+# If true disable anonymous ciphers and use self-signed certificates
+omero_server_selfsigned_certificates: false
+
 
 ######################################################################
 # Expert users only!
 ######################################################################
 
 omero_server_ice_version: "3.6"
 
+omero_server_python_requirements_ice_package:
+  RedHat: "https://github.com/ome/zeroc-ice-py-centos7/releases/download/\
```

Comment:
How to you see handling c8 and ubuntu 20.04
Redhat8?

if this is an option like that, better to introduce the convention now

Reply:
The top level variable `omero_server_python_requirements_ice_package` will be the same, the keys will change but as you can see this is `Expert users only!` so we can do that later

Reply:
as long as it does not require a major version bump for that adjustment

Reply:
It won't be. The public "API" is in the README, this is considered internal.